dev_container

```diff
@@ -1033,6 +1033,7 @@ build_and_run() {
 
     if [[ -z "${docker_file}" ]]; then
         docker_file=$(./run get_app_dockerfile ${app_name} ${app_language:-cpp})
+        c_echo "Using Docker file: ${docker_file}"
```

Comment:
Nitpick: Can we remove this to avoid extra noise? Recommend existing paths to inspect Dockerfile in use:
1. Run with `--dryrun` and inspect `./dev_container build` command output.
2. Run without `--dryrun` and inspect `docker build` command printout
3. Run `./run get_app_dockerfile` directly with the app name and inspect the output string